plugins/topology/src/components/Topology/TopologyComponent.tsx

```diff
@@ -10,23 +10,60 @@ import { ModelsPlural } from '../../models';
 import { ModelsPlural as TektonModels } from '../../pipeline-models';
 import { TopologyWorkloadView } from './TopologyWorkloadView';
 
+import '@patternfly/react-core/dist/styles/base.css';
+import '@patternfly/patternfly/patternfly-theme-dark.css';
+import '@patternfly/patternfly/patternfly-charts-theme-dark.css';
+import '@patternfly/patternfly/utilities/Accessibility/accessibility.css';
 import './TopologyComponent.css';
 
 const THEME_DARK = 'dark';
 const THEME_DARK_CLASS = 'pf-v5-theme-dark';
 
+const savedStylesheets = new Set<HTMLLinkElement>();
+
 export const TopologyComponent = () => {
   const {
     palette: { type },
   } = useTheme();
+
   React.useEffect(() => {
     const htmlTagElement = document.documentElement;
+
+    const scalprumStyles = Array.from(
+      document.querySelectorAll('link[rel="stylesheet"]'),
+    ).filter(link =>
+      link.attributes
+        .getNamedItem('href')
+        ?.value?.includes('backstage-plugin-topology'),
+    );
+
+    scalprumStyles.forEach(link =>
+      savedStylesheets.add(link as HTMLLinkElement),
```

Comment:
Tested the changes, it works fine. However I have a small suggestion to replace `useEffect` with `useLayoutEffect` to add the style sheets before the browser paint, so we could make sure style always loads before the element renders in the DOM to avoid any potential flickering due to styles in future. Let me know what you think about it?

Reply:
I have updated it to use `useLayoutEffect` and tested working good. Thanks for the feedback 👍